opentelemetry-user-events-logs/examples/README.md

```diff
@@ -0,0 +1,101 @@
+# Getting Started with OpenTelemetry Rust Logs in 5 Minutes - Exporting logs to user_events
+
+## Prerequisites
+
+* Rust and Cargo: [Install](https://doc.rust-lang.org/cargo/getting-started/installation.html) them if you haven't.
+* A Linux system with [user_events](https://docs.kernel.org/trace/user_events.html) enabled: Ubuntu 23.10 (once available), or kernel 6.4 or later build with user_events support.
+* Correct Permissions: Linux user(preferable non-root) with write permission to the tracefs.
+* A listener/agent to listen for new events. Or else [perf](https://perf.wiki.kernel.org/index.php/Main_Page) and [decode-perf](https://github.com/microsoft/LinuxTracepoints/tree/main/libeventheader-decode-cpp/tools) tools installed for validation.
```

Comment:
What does "or else" mean?

Reply:
Since this is more of a getting-started example, we don't have to list all possible ways, I suggest that we just provide the most common way or an easy way (e.g. download XYZ tool and use that).

Reply:
Good point. One documenting the perf/decode-perf tool now.